gradle-server/src/main/java/com/github/badsyntax/gradle/handlers/GetDependenciesHandler.java

```diff
@@ -122,7 +122,7 @@ private void createTemplateScript(File pluginFile, File outputFile) throws IOExc
             + "allprojects {\n"
             + "    apply plugin: com.microsoft.gradle.GradlePlugin\n"
             + "}\n";
-    Files.writeString(outputFile.toPath(), template);
+    Files.write(outputFile.toPath(), template.getBytes());
```

Comment:
Here to support Java 8 runtime. (`Files.writeString()` has been supported since Java 11)